readme.txt

```diff
@@ -81,6 +81,10 @@ Follow these instructions (https://www.postnl.nl/Images/aanvragen-api-key-stappe
 
 == Changelog ==
 
+= 5.8.1 (2025-09-06) =
+* Add: Separate fees for Home delivery and Pick-up point delivery.
+* Tweak: Show fee amounts on checkout tabs.
```

Comment:
```suggestion
* Add: Option to apply extra fees for Home Delivery and Pickup Point Delivery.
```

And lets add it to `### 5.8.0 (2025-xx-xx) =` section since its not released yet